homeassistant2influxdb.py

```diff
@@ -211,7 +212,7 @@ def main():
                         event = Event(
                             _event_type,
                             data={"new_state": state},
-                            time_fired=_time_fired
+                            time_fired=datetime.strptime(_time_fired, "%Y-%m-%d %H:%M:%S.%f")
```

Comment:
I've had to revert that line to have the script working with MariaDB. Might be worth wrapping in a condition based on the chosen DB

Reply:
Thanks for the feedback. Will update my PR tomorrow.